app/Repositories/MessageRepository.php

```diff
@@ -120,6 +120,12 @@ public function updateMessagesForContest($contest, $messages)
                 $data['type'] = $type;
                 $data['key'] = $key;
 
+                if ($data['type'] === 'leaderboard') {
+                    if (! array_key_exists('show_images', $data)) {
+                        $data['show_images'] = 0;
```

Comment:
@sbsmith86 you can use a mutator! set it to something if not set! we do this already for other fields in Gladiator forms. Gimme a sec to find an example 😄